airflow/www/views.py

```diff
@@ -756,13 +758,13 @@ def index(self):
                 current_dags = all_dags
                 num_of_all_dags = all_dags_count
 
-            dags = (
-                current_dags.order_by(DagModel.dag_id)
-                .options(joinedload(DagModel.tags))
-                .offset(start)
-                .limit(dags_per_page)
-                .all()
-            )
+            sort_column = DagModel.__table__.c.get(arg_sorting_key)
+            if sort_column is not None:
+                if arg_sorting_direction == 'desc':
+                    sort_column = desc(sort_column)
```

Comment:
```suggestion
                    sort_column = sort_column.desc()
```

I think you can do this instead?

Reply:
Done :)